src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx

```diff
@@ -19,6 +21,51 @@ import { ACCOUNT_TYPES_OPTIONS } from '../constants';
 
 import PaymentInformationEditModalError from './PaymentInformationEditModalError';
 
+const useNewForm = !environment.isProduction();
+
+const schema = {
```

Comment:
A lot of the schema and uiSchema values are pulled out from the existing form's configuration.